packages/fxa-auth-client/test/client.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as assert from 'assert';
+import AuthClient from '../server';
+
+describe('lib/client', () => {
+  let client: AuthClient;
+
+  before(() => {
+    client = new AuthClient('http://localhost:9000', {
+      timeout: 3e3,
+      keyStretchVersion: 2,
+      requireHeaders: true,
+    });
+  });
+
+  it('creates default client', () => {
+    client = new AuthClient('http://localhost:9000');
```

Comment:
I think I'm confused by the `before()` block versus this line&hellip; why are we setting `client` in a `before()` block above on L12 if we're seemingly immediately overwriting it on L20 here, and L25 below? What is L12-16 doing?